source/ore/publish.rst

```diff
@@ -145,3 +145,57 @@ Support
 -------
 
 Support cannot do anything, merely a way of showing that the member is a part of the project.
+
+
+Documenting Your Project With Pages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To document your plugin Ore offers the creation of pages, by default you get a 'Home' page when you create your new Project.
```

Comment:
You seem to use a comma where a full stop (or semicolon) belongs. Two separate sentences make more sense.
"To document your plugin Ore offers the creation of pages. By default you get a 'Home' page when you create your new Project." 
This happens in a few places ...